components/cards/UserAvailability.tsx

```diff
@@ -21,46 +21,43 @@ export default function UserAvailability({
       <CustomText style={styles.name} fontWeight="semibold">
         {fullName}
       </CustomText>
-      <View style={styles.header}>
-        <CustomText
-          fontFamily="writer-monos"
-          numberOfLines={2}
-          ellipsizeMode="tail"
-          style={styles.status}>
-          {activity}
-        </CustomText>
-        <View style={styles.dot} />
-        <CustomText style={styles.time} fontFamily="writer-monos">
-          {time}
-        </CustomText>
-      </View>
+
+      <CustomText style={styles.header} fontFamily="writer-monos">
+        <Text style={{ color: "rgba(0, 0, 0,1)" }}>{activity}</Text>
+        <Text
+          style={{
+            ...styles.dot,
+            fontFamily: "System",
+            marginVertical: "auto",
+          }}>
+          {" \u2022 "}
+        </Text>
+        <Text style={styles.time}>{time}</Text>
+      </CustomText>
     </View>
   )
 }
+
 const styles = StyleSheet.create({
   container: {
     gap: 1,
   },
   header: {
     flexDirection: "row",
-    gap: 6,
   },
   name: {
     maxWidth: "70%",
   },
   time: {
-    opacity: 0.5,
     textTransform: "capitalize",
-  },
-  status: {
-    maxWidth: "80%",
+    // Opacity itselt cannot work here due to text nesting
+    color: "rgba(0, 0, 0,0.5)",
```

Comment:
in the colors configured in the theme.ts file `rgba(0, 0, 0,0.5)` corresponds to `theme.colors.black_500` , 
```suggestion
    color: theme.colors.black_500,
```